docker-compose.yml

```diff
@@ -1,9 +1,10 @@
 version: '2'
 services:
   database:
-    image: postgres
+    image: postgres:9.6.12
```

Comment:
Why are we using such an old version of Postgres?
The current version is 12.2
9.6 originally came out in 2016

Reply:
yeah, but this is the version in AWS, we should use the same in Docker. Until we upgrade the DB